_posts/2024-07-16-Lumi_web_introduction.md

```diff
@@ -0,0 +1,219 @@
+# Before you begin
+
+You must be assigned to a LUMI project before using any of its features. Follow these [instructions](https://docs.csc.fi/accounts/how-to-create-new-project/#how-to-create-finnish-lumi-projects) to apply for and gain access to a LUMI project.
+
+# Using LUMI web interface
+
+After gaining access to your LUMI project, here is how to submit a quantum circuit to the Helmi QPU. This guide introduces two main methods for submitting jobs:
+
+1. **Interactive Session using Jupyter Notebook**: This method allows you to develop and test your quantum circuits interactively, providing a flexible and user-friendly environment for iterative experimentation.
+
+2. **Built-in Terminal on LUMI Web Interface**: This method offers a command-line approach for submitting quantum circuits, suitable for users who prefer or require terminal-based interaction.
+
+Additionally, you have the option to SSH into LUMI locally. By doing so, you can use the same commands as detailed in the [terminal instructions](#terminal-access) section to submit jobs directly from your local environment.
+
+## Interactive access with Jupyter Notebook
+
+Using an Jupyter Notebook, you can interactively develop and test your quantum circuits, making it easier to iterate and refine your experiments.
+
+### Create a Jupyter session by LUMI web interface
+
+Login into LUMI [web interface](https://www.lumi.csc.fi/public/) then click on `Jupyter` to open an interactive session
+
+<div style="text-align: center;">
+    <figure style="display: inline-block; text-align: left;  margin: 0; padding: 0;">
+        <img src="/assets/images/helmi-lumi-web-interface/jupyter_starting.png" alt="jupyter start">
+    </figure>
+</div>
+
+Fill in the required fields as shown in the picture below:
+
+<div style="text-align: center;">
+    <figure style="display: inline-block; text-align: left;  margin: 0; padding: 0;">
+        <img src="/assets/images/helmi-lumi-web-interface/lumi_jupyter.png" alt="jupyter on lumi">
+    </figure>
+</div>
+
+1. Choose the project id that you have assigned for.
+2. Select the partition that connects to Helmi. Currently, only the `q_fiqci` partition is available for this purpose.
+3. Add script to configure the quantum software stack environment:
+
+```bash
+module use /appl/local/quantum/modulefiles 
+module load helmi_qiskit 
+# or module load helmi_cirq 
+# if you want to use Cirq instead of Qiskit
+```
+
+Then click `Launch` to start the Jupyter session.
+
+### Example notebook code
+
+Once your Jupyter session is active, you can start writing and testing your quantum circuits directly within the notebook. Below is an example use of code that you can run in your Jupyter Notebook to interact with the Helmi QPU.
+
+<div style="text-align: center;">
+    <figure style="display: inline-block; text-align: left;  margin: 0; padding: 0;">
+        <img src="/assets/images/helmi-lumi-web-interface/jupyter_example.png" alt="jupyter example">
+    </figure>
+</div>
```

Comment:
Is it possible that this jupyter notebook could be added to FiQCI examples page and a link added? So that users can download this notebook and then directly use this example on LUMI.

Reply:
yes, i will make a PR to Fiqci example